### DISCUSSION TITLE
Trigger events form Client to Server

### DISCUSSION THREAD

**Discussion by naresh-io** [soketi/pws#366] [Q&A]
Hi 👋

How can we configure Soketi to send events from client to server? I can’t find any documentation on this.
I'm using Soketi with Laravel Echo.

Any thoughts on how to achieve this would be greatly appreciated.

**Comment by rennokki** [comment]
Client -> Server broadcasting is not possible, and is not supported by Pusher either. You can use [webhooks](https://docs.soketi.app/advanced-usage/app-webhooks) at most to detect messages or channel joins/leaves.

What is your use case?

**Reply by naresh-io** [reply]
Hey thanks for the for the reply, I'm building a chat application.

**Comment by lorenzoaiello** [comment]
@naresh-io - are you talking about Client Events? https://pusher.com/docs/channels/using_channels/events/#triggering-client-events

**Reply by naresh-io** [reply]
Nope, my requirement is client to server

**Reply by rennokki** [reply]
Make your users send messages via a HTTP endpoint on your server and do whatever with the message and broadcast the event on the chat room.

**Reply by naresh-io** [reply]
I'm using AJAX at the moment and I wanted to know if the alternative is possible. Thank you for the clarification.

**Reply by lorenzoaiello** [reply]
I may be misunderstanding client events, but if you enable client events, are they not captured and sent via batched webhooks?

ref: https://pusher.com/docs/channels/server_api/webhooks/#client-events

**Reply by naresh-io** [reply]
Not sure, but I'm new to WebSockets and pusher. I'll check the reference you shared, Thank you.
